index.js

```diff
@@ -0,0 +1,150 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+const statusColors = {
+  'success': 'lightgreen',
+  'fail': 'indianred',
+  'loading': 'sandybrown'
+}
+
+const reportStatus = (message, status) => {
+  let color = statusColors[status];
+  return $('#status-message').show(1000).html(message).css('background-color', color);
+};
+
+const clearContent = (idArr) => {
+  idArr.forEach((id) => {
+    $(`#${id}`).empty();
+  });
+};
+
+const reservationForm = ['reservation-name', 'email'];
+const tripForm = ['name', 'continent', 'category', 'weeks', 'cost', 'about'];
+
+const clearForm = (form) => {
+  if (form === 'reservation'){
+    reservationForm.forEach((formField) => {
+      $(`input[name="${formField}"]`).val('');
+    });
+  } else if (form === 'trip') {
+    tripForm.forEach((formField) => {
+      $(`input[name="${formField}"]`).val('');
+    });
+  }
+};
+
+const getFormData = (form) => {
+  let data = {};
+  if (form === 'reservation') {
+    data['name'] = $('input[name="reservation-name"]').val();
+    data['email'] = $('input[name="email"]').val();
+  } else if (form === 'trip'){
+    tripForm.forEach((formField) => {
+      data[formField] = $(`input[name="${formField}"]`).val();
+    });
+  }
+  return data;
```

Comment:
This is really awesome!

I'm sure there's a step even further than this to abstract it completely so it works with any form you give it (instead of checking if it's the reservation form or trip form), but I really like what's going on here.

Also instead of checking if `form` is reservation or trip in if statements, if this function got bigger I might recommend making it a case/switch statement?